<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QDialog" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual File Name Fixer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layoutPath">
     <item>
      <widget class="QLabel" name="lblPathToCheck">
       <property name="text">
        <string>Path to Check:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPathLine">
       <item>
        <widget class="QLineEdit" name="lePath"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnPath">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutPreviousName">
     <item>
      <widget class="QLabel" name="lblPreviousName">
       <property name="text">
        <string>Previous Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePreviousName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblNewName">
       <property name="text">
        <string>New Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leNewName"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>Status: Good!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnApplyAndFindNext">
       <property name="text">
        <string>Apply and Find Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
